packages/create-remix/templates/_shared_ts/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "resolveJsonModule": true,
     "target": "ES2019",
     "strict": true,
+    "baseUrl": ".",
```

Comment:
This should also be changed in the basic example

Reply:
Thanks Michael!

I changed the files: https://github.com/remix-run/remix/pull/482/commits/41fe1de30f92755711390ee26abb00f308360f24